packages/@okta/vuepress-site/docs/guides/oin-sso-overview/index.md

```diff
@@ -80,6 +104,6 @@ Ready to get started? Choose how you want to implement your SSO integration:
 </Cards>
 <br>
 
-After your app integration is built and tested, [submit the integration](/docs/guides/submit-app/) to the Okta OIN team for verification through the OIN Manager.
+After your app integration is built and tested, [review the submission requirements](/docs/guides/submit-app-prereq/main/) and [submit the integration](/docs/guides/submit-app/) to the Okta OIN team for verification.
```

Comment:
we have some inconsistency throughout with "App integration" v "integration" v "application"